src/pages/HistoryPage/HistoryPage.tsx

```diff
@@ -3,6 +3,7 @@ import { useState, useMemo, useEffect } from 'react';
 import { isUserAdmin } from 'lib/users';
 import { useQuery } from 'react-query';
 
+import isFeatureEnabled from '../../config/features';
```

Comment:
Can we just import it like

`import isFeatureEnabled from 'config/features';` ?

Reply:
uhh i dunno, this is what my editor did?

Reply:
seems to work, maybe something we can tweak w/ prettier config?